lib/rules/space-infix-ops.js

```diff
@@ -27,33 +27,50 @@ module.exports = function(context) {
                 OPERATORS.indexOf(op.value) >= 0 &&
                 (tokens[i - 1].range[1] >= op.range[0] || op.range[1] >= tokens[i + 1].range[0])
             ) {
-                return false;
+                return op;
             }
         }
-        return true;
+        return null;
     }
 
-    function report(node) {
-        context.report(node, "Infix operators must be spaced.");
+    function report(mainNode, culpritNode) {
+        var location = {
+            line: culpritNode.loc.start.line,
+            column: culpritNode.range[0]
+        };
+
+        context.report(mainNode, location, "Infix operators must be spaced.");
```

Comment:
Actually, you can just pass `culpritNode.loc.start` as the second argument here.